.github/workflows/release.yml

```
@@ -123,7 +123,7 @@ jobs:
           process_gcloudignore: false
 
   status:
-    if: always()
+    if: inputs.dry-run == null || !inputs.dry-run
```

```suggestion
    if: inputs.dry-run != null && !inputs.dry-run
```
The `null` value is considered as dry-run mode also.

---

done!